nipype/interfaces/base/core.py

```diff
@@ -276,7 +276,16 @@ def _check_version_requirements(self, trait_object, raise_exception=True):
             version = LooseVersion(str(self.version))
             for name in names:
                 min_ver = LooseVersion(str(trait_object.traits()[name].min_ver))
-                if min_ver > version:
+                try:
+                    too_old = min_ver > version
+                except TypeError:
+                    iflogger.warning(
+                        f"Nipype cannot validate the package version {version!r} for "
+                        f"{self.__class__.__name__}. Trait {name} requires version "
+                        f">={min_ver}. Please verify validity."
+                    )
```

Comment:
should this and the next one be a warning or an exception if you can't parse? i feel warnings are one of those things that people, especially users never see in complex workflows. and since the underlying tool could be installed separately, could cause major headaches for debugging.

this almost sounds like we should add a config option that allows a specific interface or all interfaces to work when there is a version mismatch or not. we already have this execution variable `stop_on_unknown_version` which we could use here i think.

Reply:
thanks @effigies. LGTM.